presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -14,7 +14,90 @@ make it executable with ``chmod +x``, then run it:
 
     ./presto --server localhost:8080 --catalog hive --schema default
 
-Run the CLI with the ``--help`` option to see the available options.
+The available options for the CLI are:
```

Comment:
@electrum the same thing is applicable to function list and other commands that are supported. If someone is updating an option then they should be updating the docs as well.